app/views/api/v2/hosts/main.json.rabl

```diff
@@ -33,6 +33,8 @@ end
 # display the token, if it hasn't expired
 node(:token, :if => ->(h) { h.token && !h.token_expired? }) { |host| host.token.value }
 
+node(:registration_token) { |h| h.registration_token }
```

Comment:
Should we have a test for this attribute being present? Probably at https://github.com/theforeman/foreman/blob/2801602393aee2245a2f5066d3dc4e731fea40e2/test/controllers/api/v2/hosts_controller_test.rb